Comp2023/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -233,11 +235,19 @@ public void updateSwerveOdometry( )
       Pose2d botLLPose = RobotContainer.getInstance( ).m_vision.getLimelightValidPose(getPose( ));
       double latency = RobotContainer.getInstance( ).m_vision.getTargetLatency( );
 
-      if ((botLLPose != null))
+      if (botLLPose != null && !DriverStation.isAutonomous( ))
```

Comment:
This doesn't cover the case quite right. There are at least four states of the robot:  Disabled, Teleop, Autonomous, and Test. Since periodic will run in disabled mode, the pose estimator will kick in here. I think you want it to use the positive case (when isTeleop is true), right?